### PR TITLE
posix: handle sway launch option

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: posix/init/src/stage2.cpp [hunk 1]
```diff
@@ -189,6 +189,12 @@ int main() {
 				throw std::runtime_error("dup2() failed");
 			close(fd);
 			execl(command.c_str(), command.c_str(), nullptr);
+		}else if(launch == "sway") {
+			setenv("WLR_DRM_NO_MODIFIERS", "1", 1);
+			setenv("WLR_BACKENDS", "drm,libinput", 1);
+			setenv("WLR_RENDERER_ALLOW_SOFTWARE", "1", 1);
+			setenv("WLR_XWAYLAND", "/usr/bin/Xwayland", 1);
```
Comment (ArsenArsen) [inline posix/init/src/stage2.cpp:196]: why is this needed?

Comment (no92) [reply posix/init/src/stage2.cpp:196]: I can't put my finger on why, but sometimes sway picked up different settings for those. This sets what works best for managarm.

**Review by avdgrinten — APPROVED** [review]
LGTM but we should wait until Sway is merged before adding this.

**Review by Dennisbonke — APPROVED** [review]
LGTM
